eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -96,6 +97,7 @@ library
         cardano-slotting,
         cardano-strict-containers,
         containers,
+        cuddle,
```

Comment:
`cardano-ledger-binary` version above needs to be updated to >= 1.3.4  (if  [this suggestion is followed](https://github.com/IntersectMBO/cardano-ledger/pull/4437/files#r1662738083))

Reply:
It already is in testlib. Should one update the version when the testlib library is updated but nothing else?

Reply:
Oh, sorry, you mean that Conway depends on Cuddle. I can move to testlib, I had placed it in the library merely to reflect the current location of the CDDL

Reply:
I'll be a bit more precise. This is where it needs to go: https://github.com/IntersectMBO/cardano-ledger/blob/8cff9cfcf23686bbde8bfa5dc0682630af38c517/eras/conway/impl/cardano-ledger-conway.cabal#L112

Reply:
Current location of CDDL is in [the package](https://github.com/IntersectMBO/cardano-ledger/blob/8cff9cfcf23686bbde8bfa5dc0682630af38c517/eras/conway/impl/cardano-ledger-conway.cabal#L16), not the library. So, sublibraries have full access to those files as well